### ISSUE TITLE
Addition for marker

### ISSUE THREAD

**Issue by Limonlol** [icerockdev/moko-maps#6]
Need add property "isAnimated" for func `move(position: LatLng, rotation: Float)`

**Comment by Alex009** [comment]
will be available in 0.2.0 via:
```kotlin
interface Marker : MapElement {
    var position: LatLng
    var rotation: Float

    fun move(position: LatLng, rotation: Float, duration: Duration)
}
```
`move` always with animation, `duration` used to control animation's duration.
`position` and `rotation` work without animation

**Comment by Alex009** [comment]
released in 0.2.0
